package.json

```diff
@@ -39,7 +39,7 @@
     "serialize-error": "^8.0.1",
     "simple-git": "^3.5.0",
     "strip-ansi": "^6.0.1",
-    "ts-jest": "^27.1.2",
+    "ts-jest": "^28.0.0-next.2",
```

Comment:
Not sure if we want to wait for stable here.

We also need to wait for a fix to the Jest 28 types as majority of our repos use `"skipLibCheck": false`.